runtime/rasdump/javadump.cpp

```diff
@@ -90,6 +93,9 @@ static UDATA innerMemCategoryCallBack (U_32 categoryCode, const char * categoryN
 static jvmtiIterationControl heapIteratorCallback   (J9JavaVM* vm, J9MM_IterateHeapDescriptor*   heapDescriptor,    void* userData);
 static jvmtiIterationControl spaceIteratorCallback  (J9JavaVM* vm, J9MM_IterateSpaceDescriptor*  spaceDescriptor,   void* userData);
 static jvmtiIterationControl regionIteratorCallback (J9JavaVM* vm, J9MM_IterateRegionDescriptor* regionDescription, void* userData);
+#if JAVA_SPEC_VERSION >= 21
+static jvmtiIterationControl continuationIteratorCallback (J9VMThread *vmThread, J9MM_IterateObjectDescriptor *object, void* userData);
```

Comment:
No space before `(` and space before, not after `*`, please (in all new or modified code).